packages/main/test/pages/Input.html

```diff
@@ -427,14 +434,27 @@ <h3>Inputs with clear icon</h3>
 
 	<h3>Input with open suggestions on focusin</h3>
 	<ui5-input id="openPickerInput" show-suggestions show-clear-icon class="input3auto" placeholder="Click to show recent values...">
-		<ui5-suggestion-group-item>Recent Items</ui5-suggestion-group-item>
-		<ui5-suggestion-item type="Inactive">No recent users</ui5-suggestion-item>
+		<ui5-suggestion-item-group header-text="Recent Items">
+			<ui5-suggestion-item type="Inactive">No recent users</ui5-suggestion-item>
+		</ui5-suggestion-item-group>
+	</ui5-input>
+
+	<br>
+	<br>
+
+	<h3>Input with openPicker() on focusin</h3>
```

Comment:
You can remove this sample because it is from an old code - probably left after some merge conflicts. 
openPicker() is no longer present in the input API